tests/Functional/Anonymizer/FrFR/AddressAnonymizerTest.php

```diff
@@ -72,7 +72,7 @@ public function testAnonymize(): void
 
         $anonymizator = new Anonymizator(
             $this->getConnection(),
-            new AnonymizerRegistry(),
+            new AnonymizerRegistry(''),
```

Comment:
Pourquoi t'as pas mis null en valeur par défaut sur le paramètre $projectDir ?

Reply:
ouais. Bonne idée. But please speak in english on this community project ^^

Reply:
Oh sorry, I forgot.